client/curriculum/css/BoxModel.tsx

```diff
@@ -14,6 +14,7 @@ import Highlight from '~/components/Highlight/Highlight';
 import DevToolsClone from '~/components/DevToolsClone';
```

Comment:
A lot of changes in this file, many unrelated to `<FormattedText>`. I think you re-formatted the entire file.

Please revert everything except the actual `<FormattedText>` code.  There should be just a few, specific changes to this file, where you replace the old with the new.